neqo-transport/src/recovery/mod.rs

```diff
@@ -836,11 +833,7 @@ impl LossRecovery {
                 if t <= now {
                     qdebug!([self], "PTO timer fired for {}", pn_space);
                     let space = self.spaces.get_mut(*pn_space).unwrap();
-                    lost.extend(
-                        space
-                            .pto_packets(PtoState::pto_packet_count(*pn_space))
-                            .cloned(),
-                    );
+                    lost.extend(space.pto_packets().cloned());
```

Comment:
Do we still need pto_packet_count if this is the decision?

The other question I have is whether this is necessary.  We're cloning all of the information so that we can process the loss, which means more work on a PTO.  Maybe PTO is rare enough that this doesn't matter, but one of the reasons for the limit on number was to avoid the extra work.

Reply:
> Do we still need pto_packet_count if this is the decision?

We do still need it to limit the number of packets we send on PTO.

> The other question I have is whether this is necessary. We're cloning all of the information so that we can process the loss, which means more work on a PTO. Maybe PTO is rare enough that this doesn't matter, but one of the reasons for the limit on number was to avoid the extra work.

I've been wondering if it would be sufficient to mark _n_ packets per space as lost, instead of all.